### PR TITLE
Image Hosting for Private Repos

### LINKED ISSUE #509: Private image hosting docs

### PR DESCRIPTION
**Pull request by heitortsergent**
Closes #509.

- [x] Add images to web projects via + -> Add image
- [x] File size limit

### CODE REVIEW

File: 4.-documentation/f.adding-images.md [hunk 1]
```diff
@@ -30,6 +27,20 @@ You can add multiple images to any documentation file. Supported image formats i
 
 To customize images, such as adding backgrounds, captions or focus, check out [images in Stoplight Flavored Markdown](stoplight-flavored-markdown.md).
 
+## Image Visibility
+
+Images that are uploaded to Stoplight via the drag-and-drop method will have a image URL such as: https://stoplight.io/api/v1/projects/cHJqOjI/images/f411rdRcvGA.
+
+The image URL has the same visibility as the Stoplight project visibility, so images uploaded to a public project will be visible by anyone who has access to the image URL, while images uploaded to internal or private projects will only be visible by users who have access to those projects in Stoplight.
```
Comment (pamgoodrich) [inline 4.-documentation/f.adding-images.md:34]: I thought someone said that project visibility didn't impact image visibility. Did that get fixed?

File: 4.-documentation/f.adding-images.md [hunk 2]
```diff
@@ -30,6 +27,20 @@ You can add multiple images to any documentation file. Supported image formats i
...
+
+For private Git repositories that are connected to a public Stoplight project, Stoplight will display a public copy of any images in the repository that are included in a Markdown article or API documentation. That way users can view a project's documentation as it was intended without having access to a Stoplight project, and Stoplight users can still use a private Git repository.
```
Comment (pamgoodrich) [inline 4.-documentation/f.adding-images.md:36]: Oh this is what I was thinking of. So if you have a private Git repo and an internal project, your images are not public, correct? The first paragraph sounds like that's the case.

File: 4.-documentation/f.adding-images.md [hunk 3]
```diff
@@ -30,6 +27,20 @@ You can add multiple images to any documentation file. Supported image formats i
...
+
+## Limitations
+
+Stoplight supports uploading images with a file size of up to 15mb.
+
+If you have an image in a Git repository that's larger than 15mb, that image will not be displayed in your Stoplight project.
```
Comment (pamgoodrich) [inline 4.-documentation/f.adding-images.md:42]: 15mb > 15 MB

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/bits-bytes-terms

**Review by pamgoodrich — CHANGES_REQUESTED** [review]
I asked a couple of questions about access because we seem to have conflicting guidance.

File: 4.-documentation/f.adding-images.md (hunk 1 above)
Comment (heitortsergent) [reply 4.-documentation/f.adding-images.md:34]: Yeah, I think I said that to you. Then I tested it and found out drag-and-drop does actually follow project permissions. :D

File: 4.-documentation/f.adding-images.md (hunk 2 above)
Comment (heitortsergent) [reply 4.-documentation/f.adding-images.md:36]: Yep, that's correct.

File: 4.-documentation/f.adding-images.md (hunk 3 above)
Comment (heitortsergent) [reply 4.-documentation/f.adding-images.md]: Fixed it, thanks!

**Review by pamgoodrich — APPROVED** [review]

**Pull request merged**